examples/tcp_client.rs

```diff
@@ -75,16 +76,18 @@ fn create_client(rx: mpsc::Receiver<Packet>, tx: mpsc::Sender<Packet>) -> IoFutu
         }
     };
 
+    let stats = Stats::new();
+
     let client = TcpStream::connect(&addr)
         .map_err(Error::from)
         .and_then(move |socket| {
             make_client_handshake(socket, &client_pk, &client_sk, &server_pk)
                 .map_err(Error::from)
         })
-        .and_then(|(socket, channel)| {
+        .and_then(move |(socket, channel)| {
             debug!("Handshake complited");
 
-            let secure_socket = Framed::new(socket, codec::Codec::new(channel));
+            let secure_socket = Framed::new(socket, codec::Codec::new(channel, stats.clone()));
```

Comment:
No need to clone stats?

Reply:
Does not matter. You will remove this file in your PR